integration-cli/trust_server.go

```diff
@@ -106,7 +106,7 @@ func newTestNotary(c *check.C) (*testNotary, error) {
 	}
 
 	// Wait for notary to be ready to serve requests.
-	for i := 1; i <= 5; i++ {
+	for i := 1; i <= 20; i++ {
```

Comment:
I thought more attempts was better than a longer "sleep", so that we don't make the tests take longer if it *does* come up fast enough